app/src/main/java/com/github/wnder/user/GuestUser.java

```diff
@@ -9,38 +9,8 @@
  */
 public class GuestUser extends User{
 
-    /**
-     * get the name
-     * @return "Guest"
-     */
-    @Override
-    public String getName(){
-        return "Guest";
-    }
-
-    /**
-     * get the profile picture
-     * @return default profile picture
-     */
-    @Override
-    public Uri getProfilePicture(){
-        return Uri.parse("android.resource://com.github.wnder/" + R.raw.ladiag);
-    }
 
-    /**
-     * returns radius for current user
-     * @return radius
-     */
-    public int getRadius(){
-        return radius;
+    public GuestUser(){
+        super("Guest", Uri.parse("android.resource://com.github.wnder/" + R.raw.ladiag));
```

Comment:
They do have different field for their constructor which I think is enough of a difference.
It also make it easy to know if the user is signed in or not (whith `instanceof` though it might not be the cleanest way to do it).
Also, on my next task I'll have to add a uid to signed in user, which the guest user won't get so they will be clearly different there.

Reply:
Yes no problem, it's a more global problem so it would be hard to change it on this PR.